contracts/deploy/mainnet/111_morpho_wrap_and_transfer.js

```diff
@@ -0,0 +1,52 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "111_morpho_wrap_and_transfer",
+    forceDeploy: false,
+    // forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    proposalId:
+      "52303668283507532886105041405617076369748861896782994594114630508874108983718",
+  },
+  async () => {
+    const { strategistAddr } = await getNamedAccounts();
+
+    const cLegacyMorpho = await ethers.getContractAt(
+      [
+        "function approve(address,uint256) external",
+        "function balanceOf(address) external view returns(uint256)",
+      ],
+      "0x9994E35Db50125E0DF82e4c2dde62496CE330999"
+    );
+
+    const cWrapperContract = await ethers.getContractAt(
+      ["function depositFor(address,uint256) external"],
+      "0x9D03bb2092270648d7480049d0E58d2FcF0E5123"
```

Comment:
I doubled checked this as TG had 0x17C5b2eB6C72F86C98b8f3ACEc3ea47947aeC45a for the Morpho Wrapper contract. But Morpho's Notion doc had 0x9D03bb2092270648d7480049d0E58d2FcF0E5123 as per the deploy script 
https://morpho-labs.notion.site/New-MORPHO-token-136d69939e6d8047a73cd32a244f5ed7

Both are funded with new Morpho tokens but only 0x9D03bb2092270648d7480049d0E58d2FcF0E5123 is being used.

So the deploy script it good. No changes required